go/services/statestore/client.go

```diff
@@ -252,7 +252,7 @@ func (c *Client[K, V]) logKV(
 	if c.log.Enabled(ctx, slog.LevelDebug) {
 		all := []slog.Attr{
 			slog.String("key", string(key)),
-			slog.String("value", string(value)),
+			slog.String("value", string(value)), // Should we avoid logging values? Customers may put secrets in here
```

Comment:
@clecompt-msft would you be in favor of only logging keys for state store level operations?

 I know that the state store isn't _supposed_ to hold secrets, but I bet some customers will use it that way.

Reply:
I'm okay with that. Lets the two log commands collapse into one, too.